src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -1070,3 +1072,106 @@ def forward(
         return CausalLMOutput(
             loss=loss, logits=logits, hidden_states=outputs.hidden_states, attentions=outputs.attentions
         )
+
+
+@add_start_docstrings(
+    """
+    Hubert Model with a sequence classification head on top (a linear layer over the pooled output) for tasks like
+    SUPERB.
+    """,
+    HUBERT_START_DOCSTRING,
+)
+class HubertForSequenceClassification(HubertPreTrainedModel):
```

Comment:
Can we use a `# Copied from ...` here from Wav2Vec2 if the code is the same?

Reply:
Probably, but I couldn't figure out how to make it work with `self.hubert = ` vs `self.wav2vec2 =`. That's the only difference other than model names.

Reply:
ah yeah you can have multiple name changes seperated by comma -> this should work:

```
# Copied from transformers.models.wav2vec2.modeling_wav2vec2.Wav2Vec2ForSequenceClassification with Wav2Vec2->Hubert, wav2vec2->hubert
```

Reply:
See: https://github.com/huggingface/transformers/blob/3bb44662602b06ffd3ca25b585b5eb515bc22995/src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py#L2306